src/ReasonApollo.re

```diff
@@ -1,15 +1,38 @@
-let apolloClient = Apollo.createApolloClient;
+let createApolloClient =
+    (
+      ~cache,
+      ~link,
+      ~ssrMode=?,
+      ~ssrForceFetchDelay=?,
+      ~connectToDevTools=?,
+      ~queryDeduplication=?,
+      ()
+    ) => {
+  let apolloClientOptions = {
+    "link": link,
+    "cache": cache,
+    "ssrMode": Js.Nullable.from_opt(ssrMode),
+    "ssrForceFetchDelay": Js.Nullable.from_opt(ssrForceFetchDelay),
+    "connectToDevTools": Js.Nullable.from_opt(connectToDevTools),
+    "queryDeduplication": Js.Nullable.from_opt(queryDeduplication)
+  };
+  ApolloClient.apolloClient(apolloClientOptions)
+};
 
-module Query =
-  ReasonApolloQuery.QueryFactory(
-    {
-      let apolloClient = apolloClient;
-    }
-  );
+module type ApolloClientConfig = {let createApolloClient: ApolloClient.generatedApolloClient;};
```

Comment:
Sorry silly question: are you using `refmt`? Because it looks like many changes in this PR are not correctly formatted 🤔

Reply:
Yep, I use refmt

Reply:
I think it's because of vscode `editor.formatOnSave` was set to true